src/collector/Backend.cpp

```diff
@@ -270,9 +309,25 @@ void Backend::print_json(rapidjson::Writer<rapidjson::StringBuffer> & writer,
     writer.Key("status");
     writer.String(status_str(m_calculated.status));
 
-    // XXX
+    writer.Key("last_start");
```

Comment:
As I know, you have agreement that all "print_json" methods should have comments with json example and description.